portfolio/src/main/webapp/about.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>About Me</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="content">
+      <div class = "header">
+        <h1>Noe's STEP Portfolio</h1>
+        <p>About Me</p>
+      </div>
+
+      <div class = "menu">
+        <a href="index.html">Home</a>
+        <a class = "active" href="about.html">About Me</a>
+        <a href="projects.html">My Projects</a>
+        <a href="talents.html">Hidden Talents</a>
+      </div>
+
+      <h2>Introduction</h2>
+      <p>Hi! My name is Noe, and I am a rising junior computer science major at UCI. I am from San Diego, and I enjoy playing soccer, playing video games, and watching Youtube in my free time. My favorite soccer team is FC Barcelona 
+        because my dad is from Barcelona and they are my hometown team. Some of my favorite video games are X-Men Legends, The Simpsons Hit and Run, and Ultimate Spider-man. 
+        I also enjoy watching Youtube, and my favorite Youtubers are KSI, RunTheFUTMarket, and David Dobrik. 
+        In addition to these hobbies, I really like listening to music. A few of my favorite artists are Juice WRLD, Post Malone, Aries, and Iann Dior.  
```

Comment:
For wrapped lines, it's suggested to use 4 space indention. You can try this tool to help you auto-format:  https://github.com/stekolla/sps-clang-format